tests/data_context/test_data_context_test_yaml_config.py

```diff
@@ -11,49 +11,17 @@
 from great_expectations.core import ExpectationSuite
 from great_expectations.data_context.store import CheckpointStore
 from great_expectations.data_context.util import file_relative_path
-from tests.test_utils import create_files_in_directory
-
-
-@mock.patch(
-    "great_expectations.core.usage_statistics.usage_statistics.UsageStatisticsHandler.emit"
+from tests.core.usage_statistics.util import (
+    usage_stats_exceptions_exist,
+    usage_stats_invalid_messages_exist,
 )
-def test_empty_store(mock_emit, empty_data_context_stats_enabled):
-    # noinspection PyUnusedLocal
-    my_expectation_store = empty_data_context_stats_enabled.test_yaml_config(
-        yaml_config="""
-module_name: great_expectations.data_context.store.expectations_store
-class_name: ExpectationsStore
-store_backend:
-    module_name: great_expectations.data_context.store.store_backend
-    class_name: InMemoryStoreBackend
-"""
-    )
-    assert mock_emit.call_count == 1
```

Comment:
Deleted this because it is covered in the individual store's test file. Perhaps a bit duplicative but I wanted each primary store in the `great_expectations.yml` to have its own test.